go/ct/gen/state.go

```diff
@@ -471,12 +471,6 @@ func (g *StateGenerator) generateWith(rnd *rand.Rand, assignment Assignment) (*s
 	// Generate return data of last call
 	resultLastCallReturnData := RandomBytes(rnd, st.MaxDataSize)
 
-	// Generate return data for terminal states.
-	var resultReturnData Bytes
-	if resultStatus == st.Stopped || resultStatus == st.Reverted {
-		resultReturnData = RandomBytes(rnd, st.MaxDataSize)
-	}
-
```

Comment:
As discussed, I have a concern about removing the generation of resutlReturnData.
The returned data is used in two different contexes:
- As write buffer for a "callee" contract, used by the RETURN op. This buffer cannot be accessed by the contract again, as the execution terminates right away.  Regarding this OP, we can remove the buffer generation
- As read buffer for the "caller" contract, accessed via RETURNDATASIZE and RETURNDATACOPY.

The later is a state that would require the generation of a returned data buffer. Right?